### PR TITLE
Feat/item table updates

### PR DESCRIPTION
**Pull request by zainkai**
### Updates Made
- item table header reorder
- adding links to item keys
- hide advanced filter if not enabled
- general styling updates

### Contributing developer checklist
- [x] I've updated source branch with the latest changes from dev.
- [x] I've added/changed unit tests for components with functionality.
- [x] I've added/updated storybook for any component that I've changed.
- [x] I've reviewed each jsdoc header in regards to the changes that I've made and updated them.

### Reviewing developer checklist
- [ ] I've pulled this branch to my local machine.
- [ ] I've inspected the changes on storybook.
- [ ] I've run the test suite and all tests have passed

### CODE REVIEW

File: src/Assets/Styles/layout.less [hunk 1]
```diff
@@ -48,7 +48,6 @@ h2 {
 }
 
 .section, .section-light, .section-translucent {
-  background-color: @background-color-base;
```
Comment (alexlepinski) [inline src/Assets/Styles/layout.less]: I'm not sure if this a good idea. This is here for a reason.

Comment (alexlepinski) [reply src/Assets/Styles/layout.less]: Please review about test items page. You can also add a  line for .section {background-color: @background-color-base}

File: src/Assets/Styles/score-page.less [hunk 2]
```diff
@@ -70,9 +70,11 @@ table.item-table {
     display: block;
     margin: 0;
     padding: 0;
-    border: 0;
+    border: 1px solid #EFEFEF;
```
Comment (alexlepinski) [inline src/Assets/Styles/score-page.less]: maybe @grey-border-color: #D5D5D5;

File: src/Assets/Styles/layout.less (hunk 1 above)
Comment (danvanhorn) [reply src/Assets/Styles/layout.less]: I don't think we want the `@background-color-base` here, these classes are getting a background color applied later if needed. This is what is causing the ugly gray box around the Item table,

Comment (danvanhorn) [reply src/Assets/Styles/layout.less]: If we do want to keep that then we should change the `.section` that wraps the `<ItemTable/>` to a `.section-translucent` as I believe that class has no background color.

Comment (danvanhorn) [reply src/Assets/Styles/layout.less]: Nevermind, `section-translucent` wont work.

File: src/Assets/Styles/score-page.less [hunk 3]
```diff
@@ -102,47 +104,62 @@ table.item-table {
...
+table.item-table > thead > tr > td:first-child,
+table.item-table > tbody > tr > td:first-child {
+    width: 50px;
+    text-align: center;
+    padding: 0px;
+    margin:auto;
+}
+
 table.item-table > thead > tr > th:nth-child(2),
 table.item-table > tbody > tr > td:nth-child(2),
```
Comment (alexlepinski) [inline src/Assets/Styles/score-page.less]: let's not do this. let's add a class for each column

**Pull request merged**